.gitignore

```diff
@@ -0,0 +1,27 @@
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/build
+
+# misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+
+package-lock.json
+dist
+yarn.lock
+.env
+.idea
```

Comment:
Hi @FahdJamy great work man, so the EOL here would be fixed later too?
thanks

Reply:
Fixed